how-to/how-to-set-prometheus-in-k8s.md

```diff
@@ -0,0 +1,70 @@
+# How to monitor Kata Containers in Kubernetes clusters
+
+This document describes how to run `kata-magent` in a Kubernetes cluster using Prometheus's service discovery to scrape metrics from `kata-agent`.
```

Comment:
I still don't like the name `kata-magent` I'm afraid :smile:

Anyone else have thoughts on this?

Reply:
@jodh-intel How about `kata-monitor` or `kata-monitoring-agent`, those are the two that I could think of. Now that think about it more, I dont mind the the longer name `kata-monitoring-agent`.

@egernst @fidencio @bergwolf Please chip in with any suggestions.

Reply:
I think I prefer `kata-monitor` - it not too long, but suggests what the component is doing. I'd rather we didn't "over-load" terms to avoid confusion (in other words, we already have one agent, so adding another is bound to confuse :-)

Reply:
I’m okay with all of the suggestions above. Spelling out the M is better imo?

Reply:
I'd prefer something like kata-monitor, and then document properly what's monitoring. :-)

Reply:
+1 for `kata-monitor`

Reply:
It seems the final winner is `kata-monitor`, I have updated.